packages/devtools_app/lib/src/shared/table/table_data.dart

```diff
@@ -81,7 +84,8 @@ abstract class ColumnData<T> {
   String? getCaption(T dataObject) => null;
 
   /// Get the cell's tooltip value from the given [dataObject].
-  String getTooltip(T dataObject) => getDisplayValue(dataObject);
+  String getTooltip(T dataObject) =>
```

Comment:
add a TODO to remove redundant overrides now that this 'showTooltip' var is supported.